cypress/integration/timeDimensions.cy.js

```diff
@@ -27,7 +27,18 @@ const assertTimeDimension = (dimension) => {
         selectEventWithProgram(trackerProgram)
         openProgramDimensionsSidebar()
         const label = trackerProgram[dimension.id]
-        selectRelativePeriod({ label, period: TEST_REL_PE_THIS_YEAR })
+        if (dimension.id === DIMENSION_ID_LAST_UPDATED) {
+            // last updated doesn't seem to get bumped yearly in the SL DB so we need to select a fixed period
+            selectFixedPeriod({
+                label,
+                period: {
+                    year: '2023',
```

Comment:
Could we use current year from the Date object instead? If we did that, would it mean that we don't need to fix this every January?

Reply:
As commented in the code above, `Last updated` doesn't actually get bumped, so it will statically remain `2023` (unless Phil changes the script). That's why I had to make a special case for it (all other time dimensions are using `TEST_REL_PE_THIS_YEAR` because they actually get bumped)